uint32 session_id
time ts
Float32XYZ pos
float32 confidence
string[] gestures
